example.py

```diff
@@ -153,13 +153,13 @@ def parse_cmdline():
                             formatter_class=RawTextHelpFormatter)
     parser.add_argument(
         "dsn",
-        help="database connection string\n\n"
-             "For cockroach demo, use 'postgresql://<username>:<password>@<hostname>:<port>/bank?sslmode=require',\n"
-             "with the username and password created in the demo cluster, and the hostname and port listed in the\n"
-             "(sql/tcp) connection parameters of the demo cluster welcome message.\n\n"
-             "For CockroachCloud Free, use 'postgres://<username>:<password>@free-tier.gcp-us-central1.cockroachlabs.cloud:26257/<cluster-name>.bank?sslmode=verify-full&sslrootcert=<your_certs_directory>/cc-ca.crt'.\n"
-             "If you are using the connection string copied from the Console, your username, password, and cluster name will be pre-populated.\n"
-             "Replace <your_certs_directory> with the path to the cc-ca.cert downloaded from the Console."
+        help="""database connection string\n\n
+             For cockroach demo, use 'postgresql://<username>:<password>@<hostname>:<port>/bank?sslmode=require',\n
```

Comment:
Don't you have empty lines now? The point of having a multi-line string is that it is printed as it is, but if you have indentation on the left and \n on the right you will end up with a weird visualization...

```

positional arguments:
  dsn            database connection string
                 
                 
                              For cockroach demo, use 'postgresql://<username>:<password>@<hostname>:<port>/bank?sslmode=require',
                 
                              with the username and password created in the demo cluster, and the hostname and port listed in the
                 
                              (sql/tcp) connection parameters of the demo cluster welcome message.
                 
                 
                              For CockroachCloud Free, use 'postgres://<username>:<password>@free-tier.gcp-us-central1.cockroachlabs.cloud:26257/<cluster-name>.bank?sslmode=verify-full&sslrootcert=<your_certs_directory>/cc-ca.crt'.
                 
                              If you are using the connection string copied from the Console, your username, password, and cluster name will be pre-populated.
                 
                              Replace <your_certs_directory> with the path to the cc-ca.cert downloaded from the Console.

optional arguments:
  -h, --help     show this help message and exit
  -v, --verbose  print debug info
```

opening a MR to fix it

Reply:
Whoops. Sorry about that. Thanks for opening a PR to fix this.